### PR TITLE
Cucumber refactor - filing package

### PR DESCRIPTION
**Pull request by karthikmsqa**
# Description

This PR includes the following proposed change(s):

- Refactored filing package using submission service

## Type of change

- [x] Refactoring / Documentation

## How Has This Been Tested?

- Local test runs and github actions on PR

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: tests/src/test/resources/features/getFilingPackage.feature [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+Feature: Filing package information can be retrieved
+
+  Scenario Outline: Validate that filing package information is retrieved
+    Given valid user account "<username>":"<password>" is authenticated
+    When user submits request to get filing package information
+    Then a valid filing package information is returned
+    Examples:
+      | username | password |
+      | bobross  | changeme |
```
Comment (TayGov) [inline tests/src/test/resources/features/getFilingPackage.feature:12]: New line

**Review by TayGov — CHANGES_REQUESTED** [review]
Looks good one minor thing

File: tests/src/test/java/ca/bc/gov/open/jag/efiling/stepDefinitions/GetFilingPackageSD.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+import java.io.File;
+import java.io.IOException;
+import java.util.UUID;
+
+public class GetFilingPackageSD {
+
+    private final OauthService oauthService;
+    private final SubmissionService submissionService;
+
+    private static final String TEST_DOCUMENT_PDF = "test-document.pdf";
```
Comment (TayGov) [inline tests/src/test/java/ca/bc/gov/open/jag/efiling/stepDefinitions/GetFilingPackageSD.java:27]: Is this file deleted?

Comment (karthikmsqa) [reply tests/src/test/java/ca/bc/gov/open/jag/efiling/stepDefinitions/GetFilingPackageSD.java:27]: It's not. Moved under "resources"

**Review by TayGov — APPROVED** [review]

**Pull request merged**